src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/tests/latest/test_acr_commands.py

```diff
@@ -271,7 +271,7 @@ def test_acr_create_replication(self, resource_group, resource_group_location):
     @ResourceGroupPreparer()
     @record_only()
     def test_acr_image_import(self, resource_group):
-        '''There are nine test cases in the function.
+        '''There are nine test cases.
```

Comment:
We don't need to write all the combinations of params as tests, but please do think about the following cases and pick the important ones to cover as tests.

 - `-r` may be a registry name, login server, resource id, in the same sub or different subs.
 - `--source` may be fully qualified or not.
 - Each of `--username`, `--password`, `--repository`, `--force`, `-t` may be specified or not.